src/main/kotlin/com/jaoafa/vcspeaker/events/TitleResetEvent.kt

```diff
@@ -12,14 +13,15 @@ import dev.kord.core.entity.channel.TextChannel
 import dev.kord.core.event.user.VoiceStateUpdateEvent
 import kotlinx.coroutines.flow.count
 
-class TitleEvent : Extension() {
+class TitleResetEvent : Extension() {
     override val name = this::class.simpleName!!
 
     override suspend fun setup() {
         event<VoiceStateUpdateEvent> {
             check {
                 failIf(event.state.getMember().isBot)
-                failIf(event.old?.getChannelOrNull() == null || event.state.getChannelOrNull() != null)
+                val currentChannel = event.state.getChannelOrNull()
+                failIf(event.old?.getChannelOrNull() == null || (currentChannel != null && !currentChannel.isAfk()))
```

Comment:
この条件式だと、「残り1人のvcからafkに移動した場合」はリセットされますが、「残り1人のvcから他のafkではないvcに移動した場合」にリセットされないように見られます
（チェックアウトして試しましたが、リセットされませんでした…）

タイトルがリセットされる条件は「vcの退出、または移動によってvc人数がbotを含まずに0人になる場合」なので、そもそも以下で良いのではないでしょうか？

```suggestion
                failIf(event.old?.getChannelOrNull() == null)
```

※適当なこと言ってたらごめんなさい

Reply:
そうかもしれない…
AFK のことしか意識してなかったわ